api/v1/v1.go

```diff
@@ -13,17 +13,21 @@ import (
 // XXX add a clamp to all batches
 
 const (
+	APIVersion = 1
+
+	RoutePrefix = "/v1/"
```

Comment:
RoutePrefix should reference the APIVersion const.
```
RoutePrefix = fmt.Sprintf("/v%v", APIVersion)
```

Don't include the second `/` in the RoutePrefix.  The route should start with a `/`.

`StatusRoute = RoutePrefix + "/status/"`

You really don't need the `/` at the end of `/status/` but you can't change it in v1 since people may be expecting it at this point.